internal/services/datafactory/data_factory_dataset_binary_resource.go

```diff
@@ -134,6 +134,11 @@ func resourceDataFactoryDatasetBinary() *pluginsdk.Resource {
 							Required:     true,
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
+						"dynamic_container_enabled": {
```

Comment:
Do these new properties need to be added to the tests and docs?

Reply:
Ohh, thanks for that catch!